### DISCUSSION TITLE
Tracking from 3rd party app

### DISCUSSION THREAD

**Discussion by chalmagean** [plausible/analytics#2143] [Self-Hosted Support]
Hi, I'm using Plausible on my own site but also on a 3rd party cart SaaS that I use. And the cart has the option to add some JS code to track the checkout page and the "thank you" page (after submitting the order).

So I've added the tracking code to both my own website and also this cart SaaS, which uses a subdomain (but the sale TLD).

I can see the event tracking request going out to my server, and returning 202. But I don't see the event in my analytics.

How can I debug this?

Thank you.

**Comment by RobertJoonas** [comment]
Hi @chalmagean!

> I can see the event tracking request going out to my server, and returning 202. But I don't see the event in my analytics.
> How can I debug this?

Could you check in the request body that you're sending the event with the correct `d` attribute? It stands for "data-domain" and the string value should match the name of your Plausible site. If that doesn't match, you can change the value with the data-domain attribute in the snippet. Hope this helps!

**Comment by chalmagean** [comment]
The `d` value is `null` even though I am using the `data-domain` attribute.

```html
<script defer data-domain="mydomain.com" src="https://analytics.mydomain.com/js/plausible.js"></script>
```
